doc/integrator/upgrade_application.rst

```diff
@@ -112,10 +112,10 @@ Upgrade the main schema:
 
 .. code:: bash
 
-   ./docker-run alembic --config alembic.ini upgrade head
+   ./docker-run alembic --config=alembic.ini --name=main upgrade head
```

Comment:
Note that --config=alembic.ini should be optional

Reply:
:-)